website/docs/guides/v2-upgrade-guide.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: "helm"
+page_title: "Helm: Upgrade Guide for Helm Provider v2.0.0"
+description: |-
+  This guide covers the changes introduced in v2.0.0 of the Helm provider and what you may need to do to upgrade your configuration.
+---
+
+# Upgrading to v2.0.0 of the Helm provider
+
+This guide covers the changes introduced in v2.0.0 of the Helm provider and what you may need to do to upgrade your configuration.
+
+## Changes in v2.0.0
+
+### Changes to Kubernetes credentials supplied in the provider block
+
+We have made several changes to the way access to Kubernetes is configured in the provider block. 
+
+1. The `load_config_file` attribute has been removed.
```

Comment:
I would include what it means that the attr has been removed. Something like "From now on, the provider will not detect a kubeconfig automatically and will require explicit setting of credentials"